src/main/com/cqfn/degitx/simulator/Edge.kt

```diff
@@ -0,0 +1,32 @@
+package com.cqfn.degitx.simulator
+
+import java.util.concurrent.ThreadLocalRandom
+
+/**
+ * Represents whole network infrastructure between two Servers. Emulates delays and throughput limitations.
+ */
+class Edge {
+    constructor(from: Server, to: Server) {
+        this.from = from
+        this.to = to
+        this.state = State.ACTIVE
+    }
+
+    val from: Server
+
+    val to: Server
+
+    /**
+     * Current state of Edge
+     */
+    var state: State
+
+    /**
+     * Maximum limit in Mb/sec
+     */
+    var throughput = 0L
+
+    fun getCurrentThroughput(): Long {
+        return (this.throughput / ThreadLocalRandom.current().nextLong(1, 10))
```

Comment:
@IlyaMoskva I have some doubts here, I suppose we can't use uniform distribution for throughput value here, I think we need to investigate it more accurately.

Reply:
@g4s8 For now it was kept as stub, but yes - here must be some special formula and distribution, based on given Graph settings. I've simplified this place, as it's not the time (and task) for implementation this part.

Reply:
@IlyaMoskva OK, so let's create a ticket to not forget about it

Reply:
Issue #19 created.